config/routes.rb

```diff
@@ -1,5 +1,6 @@
 RailsContributors::Application.routes.draw do
   get 'contributors/in-time-window/:time_window' => 'contributors#in_time_window', as: 'contributors_in_time_window'
+  get 'contributors/in-week-of/:date' => 'contributors#in_week', as: 'contributors_in_week'
```

Comment:
You could add a constraint to `:date` to ensure that the date is ISO 8601 format, e,g:

``` ruby
scope 'contributors', as: 'contributors', controller: 'contributors' do
  get 'in-time-window/:time_window', action: 'in_time_window', as: 'in_time_window'
  get 'in-week-of/:date', action: 'in_week', as: 'in_week', date: /\d{4}-\d{2}-\d{2}/
end
```